webapp/app/forms/steps/lotse_multistep_flow_steps/confirmation_steps.py

```diff
@@ -65,7 +65,8 @@ def render(self, data, render_info):
             return render_template('lotse/display_filing_success.html', render_info=render_info,
                                    elster_data=render_info.additional_info['elster_data'],
                                    header_title=_('form.lotse.filing.header-title'),
-                                   tax_number_provided=data['steuernummer_exists'] == 'yes')
+                                   tax_number_provided=data.get('steuernummer_exists') and
```

Comment:
I think we should quickly talk about how and why the integration test were broken. I think in a normal run through you should not be able to get to this page without the `steuernummer_exists` and the `steuernummer` should also not be None (s. added conditional in fields.py)

Reply:
We fixed it in 7caf86a.